app/code/Magento/ConfigurableProduct/Test/Unit/Model/Product/Type/ConfigurableTest.php

```diff
@@ -394,6 +394,11 @@ public function testGetUsedProducts()
             ->willReturn('result');
 
         $this->productCollectionFactory->expects($this->any())->method('create')->willReturn($productCollection);
+        $this->salableProcessor
+            ->expects($this->once())
+            ->method('process')
+            ->with($productCollection)
+            ->will($this->returnValue($productCollection));
```

Comment:
Remove this line after applying previous change, as return values are specified in map depending on method call params.